app/herms.hs

```diff
@@ -197,6 +197,13 @@ getServingsAndConv serv convName config = (servings, conv)
           | convName  == t Str.imperial =  Imperial
           | otherwise = defaultUnit' config
 
+findRecipes :: String -> HermsReader IO ()
+findRecipes rgx = do -- TODO: add colour option here, maybe modify findMatches to just return a list of tuples where we can colorify the first part?
+    (_, recipeBook) <- ask
+    let matches = findMatches rgx recipeBook
```

Comment:
Nit: You could inline `matches`, it's only used once and it's not a terribly complicated expression. 

(This is just a small/stylistic comment, feel free to address or not as you prefer, it won't hold up merging.)

Reply:
I think I'll keep this line as is - the control flow is more visible this way I think.

Reply:
Control flow is a tricky concept in Haskell :grin: